src/main/java/storageconnector/AbstractDBConnector.java

```diff
@@ -31,8 +33,10 @@
  * 
  */
 public abstract class AbstractDBConnector {
-	
-	public enum Connector {ON_HEAP, S3};
```

Comment:
what's the purpose of this enum?

Reply:
Needed for the config, it is easier to specify which connector a node should use if we have an enum.

Reply:
We'll later do that via a class name (or even with the class loader option). I'm not sure whether it makes sense to add stuff like that to classes that we won't use in this way later. Better hardcode the instantiation in the starter class for now and add a //FIXME.